docs/getting-started.md

```diff
@@ -10,26 +10,26 @@ Preparing your code to have it effortlessly work on KintoHub is very straightfor
 
```

Comment:
Suggest adding a "Hi there - welcome to KintoHub!" at the top of the page

Reply:
cool beans!